boards/launchxl/README.md

```diff
@@ -0,0 +1,51 @@
+## Platform specific instructions
+
+### Flashing
+Download and use [uniflash](http://processors.wiki.ti.com/index.php/Category:CCS_UniFlash) to flash. Follow the guide
+[here](http://processors.wiki.ti.com/index.php/UniFlash_v4_Quick_Guide#Standalone_Command_line_tool) in order to generate
+a standalone command line tool to ease the flashing process.
+
+The standalone CLI has been extracted, set an environment variable named `UNIFLASH_CLI_BASE` in your shell profile:
+
+```bash
+$> echo UNIFLASH_CLI_BASE="<path to extracted uniflash CLI>" >> ~/.bash_profile
+$> source ~/.bash_profile
+```
+
+Now you're able to use the Makefile targets `flash` and `flash-app` in order to load the program onto the launchxl
+board.
+
+```bash
+$> make flash       # make and flash the kernel
+$> make flash-blink # make and flash the blink app
```

Comment:
This should be clarified to make sure users understand how to load the kernel and apps.
```
```bash
$> cd boards/launchxl/
$> make flash               # make and flash the kernel
```.
```
and

```
```bash
$> cd userland/examples/blink/
$> make flash TOCK_BOARD=launchxl  # make and flash the blink app
```.
```
(ignore the period at the end of the triple tick. That was just to make it not actually close the code block)